starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,54 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" type="text/css" href="C:\Users\fredrik\Downloads\lab-css-recipes-clone-master\lab-css-recipes-clone-master\starter-code\apple-pie-starter-code\css\main.css">
```

Comment:
In this case the link is pointing directly to your computer C: drive so it is not accessible from the outside. In this case to open the css in question and have it working everywhere you would need to add: **./css/main.css**

./ refers to the current folder where index.html is
css/main.css access css folder and then the main.css file